pycbc/inject/inject.py

```diff
@@ -757,13 +796,15 @@ def make_strain_from_inj_object(self, inj, delta_t, detector_name,
 
     def end_times(self):
         """Return the approximate end times of all injections.
-
-        Currently, this just assumes all ringdowns are 2 seconds long.
         """
         # the start times are the tcs
         tcs = self.table.tc
-        # FIXME: this could be figured out using the ringdown module
-        return tcs + 2
+        mfs = self.table.final_mass
```

Comment:
Does this belong in this PR? Or should this be a small separate one that can be reviewed quickly?